gdal/ogr/ogrsf_frmts/flatgeobuf/packedrtree.cpp

```diff
@@ -316,9 +316,8 @@ std::vector<SearchResultItem> PackedRTree::streamSearch(
     auto levelBounds = generateLevelBounds(numItems, nodeSize);
     uint64_t leafNodesOffset = levelBounds.front().first;
     uint64_t numNodes = levelBounds.front().second;
-    std::vector<NodeItem> nodeItems;
-    nodeItems.reserve(nodeSize);
-    uint8_t *nodesBuf = reinterpret_cast<uint8_t *>(nodeItems.data());
+    NodeItem nodeItems[nodeSize];
```

Comment:
variable length arrays are forbidden in GDAL. I'd suggest to just allocate the std::vector to nodeSize items instead of just .reserve(). Hopefully this will not affect performance in a measurable way

Reply:
Ah yes, I recall something about variable length arrays now.. reworked to your suggestion, I don't think it's a performance critical section.